### PR TITLE
adding forgerock migration

### PR DESCRIPTION
**Pull request by mark-robustelli**
The code link will not work until [the import script pull request](https://github.com/FusionAuth/fusionauth-import-scripts/pull/14) is merged.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206525978156287

### CODE REVIEW

**Review by alex-fusionauth — CHANGES_REQUESTED** [review]
Main thing is making sure we are clear on import vs. export
The json vs. csv as well

I was able to walk through and run it after I fixed my API key perms

File: astro/src/content/docs/lifecycle/migrate-users/bulk/forgerock.mdx [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+### Use the Script
+
+The output of the script will be a `.json` file:
+
+```sh title="Running the import script with the help command line switch"
+ruby ./import.rb -h
+```
+
+```shell title="Forgerock Export Script Help Results"
```
[the remark this answers is not in the document]
Comment (mark-robustelli) [reply astro/src/content/docs/lifecycle/migrate-users/bulk/forgerock.mdx:228]: Yes, copy and paste error.  Fixed

File: astro/src/content/docs/lifecycle/migrate-users/bulk/forgerock.mdx [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+Usage: import.rb [options]
+    -r APPLICATION_IDS,              A comma separated list of existing applications Ids. All users will be registered for these applications.
+        --register-users
+    -u, --users-file USERS_FILE      The exported json user data file from Forgerock. Defaults to out.csv.
```
[the remark this answers is not in the document]
Comment (mark-robustelli) [reply astro/src/content/docs/lifecycle/migrate-users/bulk/forgerock.mdx:232]: Updated.

File: astro/src/content/docs/lifecycle/migrate-users/bulk/forgerock.mdx [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+* `-u` should point to the location of the user export file you created.
+* `-f` must point to your FusionAuth instance. If you are testing locally, it will probably be `http://localhost:9011`.
+* `-k` needs to be set to the value of the API key created above.
+* `-t` should be set to the Id of the testing tenant created above.
+
+You may or may not want to use the `-m` switch, which takes the {migration_source_name} Id and uses that for the FusionAuth user Id. If you have external systems reliant on the identifier, set this. Doing so ensures imported users have the same Id as they did in {migration_source_name}. Otherwise, you can omit this switch.
+
+When you run the script, you'll see output like:
+
+```shell title="Import script output"
```
[the remark this answers is not in the document]
Comment (mark-robustelli) [reply astro/src/content/docs/lifecycle/migrate-users/bulk/forgerock.mdx:253]: added

**Comment by mark-robustelli** [comment]
Made the changes, please let me know if there is anything else.  Thanks.

**Review by alex-fusionauth — APPROVED** [review]
🛳️ it

**Pull request merged**